test/spec/features/modeling/behavior/ReplaceElementBehaviourSpec.js

```diff
@@ -535,7 +535,7 @@ describe('features/modeling - replace element behavior', function() {
         // then
         var createdEvent = elementRegistry.get(id);
 
-        expect(createdEvent).to.eql(startEvent);
+        expect(createdEvent).to.exist;
```

Comment:
The `BpmnReplace` will now always create a new `di` and `shape`. This is the only place where we checked if it's the same, all other tests just test for an existing new shape as well:
 https://github.com/bpmn-io/bpmn-js/blob/fc2201a1bcc8d28de58ac7e477bdfba51b179a3c/test/spec/features/modeling/behavior/ReplaceElementBehaviourSpec.js#L474-L491